src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} key={post.id} />
+    ))}
+    ;
```

Comment:
There is an unnecessary semicolon here. It should be removed to avoid potential syntax issues.